### PR TITLE
Features/#548 gasturbines etrago

### LINKED ISSUE #548: Integrate the gas turbines into the eTraGo tables
The gas turbines are distributed in [power_plants.non_chp_conventional.](https://github.com/openego/eGon-data/blob/dev/src/egon/data/datasets/power_plants/conventional.py). Since they are modeled as link components between CH4 and AC buses, the integration into the eTraGo tables differs from the one for other power plants.

### PR DESCRIPTION
**Pull request by fwitte**
Fixes #548.

- [x] add task to pipeline before merging into CI!

## Before merging into `dev`-branch, please make sure that

- [x] the `CHANGELOG.rst` was updated.
- [x] new and adjusted code is formated using `black` and `isort`.
- [x] the `Dataset`-version is updated when existing datasets are adjusted. 
- [x] the branch was merged into the `continuous-integration/run-everything-over-the-weekend`- branch.
- [x] the workflow is running successful in `test mode`.
- [x] the workflow is running successful in `Everything` mode.

### CODE REVIEW

**Comment by fwitte** [comment]
@IlkaCu: is it problematic, that the column "geom" will be empty here? I figured it is not necessary, since it is only for display.

File: src/egon/data/datasets/power_etrago/match_ocgt.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    # create topology column (linestring)
+    gdf = link_geom_from_buses(gdf, scn_name)
+    gdf["p_nom_extendable"] = False
+    carrier = "OCGT"
+    gdf["carrier"] = carrier
+
+    # Delete old entries
+    db.execute_sql(
+        f"""
+        DELETE FROM grid.egon_etrago_link WHERE "carrier" = '{carrier}'
```
Comment (IlkaCu) [inline src/egon/data/datasets/power_etrago/match_ocgt.py:38]: Will there be 'OCGT' components at the foreign buses? If yes, be careful to not delete them here.

Comment (fwitte) [reply src/egon/data/datasets/power_etrago/match_ocgt.py:38]: Good, point, this is related to #567. I will try to add the methods accordlingly until tomorrows CI run.

File: src/egon/data/datasets/power_etrago/match_ocgt.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        AND scn_name = '{scn_name}';
+        AND bus0 IN (
+            SELECT bus_id FROM grid.egon_etrago_bus
+            WHERE scn_name = '{scn_name}' AND country = 'DE'
+        )
+        AND bus1 IN (
+            SELECT bus_id FROM grid.egon_etrago_bus
+            WHERE scn_name = '{scn_name}' AND country = 'DE'
+        );
```
Comment (fwitte) [inline src/egon/data/datasets/power_etrago/match_ocgt.py:47]: @IlkaCu: Do you happen to know, how this can be done with only one additional query?

Comment (IlkaCu) [reply src/egon/data/datasets/power_etrago/match_ocgt.py:47]: Don't know - but as long as it works like this, I would assume its fine.

**Comment by fwitte** [comment]
@IlkaCu: The task ran successful on CI, can you quickly rereview? I adjusted the code according to your prior comments.

**Review by IlkaCu — APPROVED** [review]
Efficiency for OCGT units are not set yet. Will this be done in subsequently?

**Comment by fwitte** [comment]
We will set efficiency, capital cost and marginal cost of links in the scenario_parameters via #550 (https://github.com/openego/eGon-data/pull/550/files?diff=unified&w=0#diff-675ea4fb78613a2b84b96ab4fcc117ab7da1d9fdb40edaaf6f33542fe0f2651bR238-R246).

**Comment by IlkaCu** [comment]
> We will set efficiency, capital cost and marginal cost of links in the scenario_parameters via #550 (https://github.com/openego/eGon-data/pull/550/files?diff=unified&w=0#diff-675ea4fb78613a2b84b96ab4fcc117ab7da1d9fdb40edaaf6f33542fe0f2651bR238-R246).

The `efficiency_fixed` column for carrier = 'OCGT' in `grid.egon_etrago_link` is still empty. Could you please also add this information to the etrago-tables.

**Comment by IlkaCu** [comment]
If I remember it right, we just agreed to store the parameters centrally in the scenario_parameters and but fill the efficiency and cost columns in the etrago-tables decentrally in the technology specific tasks/datasets. 

As it is done here: https://github.com/openego/eGon-data/blob/dev/src/egon/data/datasets/heat_etrago/__init__.py#L271-L274

If you remember it differently we an discuss this tomorrow in our regular meeting.

**Comment by fwitte** [comment]
Okay, I'll add it to the agenda. Generally, I am fine with adding the data inplace. I just waited with this task because I understood it the other way, the same applies for all data from all other gas related carriers. Opening a separate issue :).

**Pull request merged**